junebug/tests/test_api.py

```diff
@@ -57,14 +57,26 @@ def test_http_error(self):
 
     @inlineCallbacks
     def test_get_channel_list(self):
+        redis = yield self.get_redis()
+        config = self.default_channel_config
+
         resp = yield self.get('/channels')
-        yield self.assert_response(
-            resp, http.INTERNAL_SERVER_ERROR, 'generic error', {
-                'errors': [{
-                    'message': '',
-                    'type': 'NotImplementedError',
-                    }]
-                })
+        yield self.assert_response(resp, http.OK, 'channels listed', [])
+
+        yield Channel(redis, {}, config, u'test-channel-1').save()
+
+        resp = yield self.get('/channels')
+        yield self.assert_response(resp, http.OK, 'channels listed', [
+            u'test-channel-1',
```

Comment:
Since we have different intuitions about this, perhaps lets leave things as
they are and make a note to check with Nic and Mark what they think (since
they'll likely be the first users other than us)?